0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml

```diff
@@ -8,6 +8,12 @@
     <Requirements>
       <Techs datatype="tokens">phase_town</Techs>
     </Requirements>
+    <History>
+        Horses were vital for the Han military in combat against the nomadic horsemen of the North. Because combat-capable horses were, for this reason, a precious good, the possession and provision of horses for individuals were strictly regulated. There was, for example, a law in place that allowed families to exempt three men from military service if they could provide a strong, healthy horse instead.
+        However, indigenous Chinese breeds were comparably small and too weak for battle. This forced the Han to resort to other means of obtaining horses for their cavalry.
+        One possibility was through trading with various nomadic peoples. With Xiognu, for example, although their relations were generally hostile, the Han sometimes engaged in trade for livestock. The Wusun, on the other hand, another nomadic people, were close allies, and when forming the marriage alliance, the dowry even included 1,000 horses. If the Han couldn't trade for them, they took desired horses by force. The general Li Guangli famously conducted two very costly and dangerous exhibitions against the faraway Greco-Bactria to get hands on powerful horses — The War of the Heavenly Horses.
```

Comment:
Could you go into a bit more detail with the marriage alliance? Not much is needed, just who married whom.

Reply:
Like this? Or do you want the names?

Reply:
That's good thanks.